### PR TITLE
Bug 1113134 - Second phone number gets duplicated if you delete the firs.t one and press "add phone" r=ferjm +autolander

### PR DESCRIPTION
**Pull request by arcturus**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[arcturus Francisco Jordano (arcturus)](https://github.com/arcturus) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=55296f1ce83a)

File: apps/communications/contacts/js/views/form.js [hunk 1]
```diff
@@ -478,6 +478,22 @@ contacts.Form = (function() {
     }
   }
 
+  /**
+   * We cannot relay on the counter, but in the next id after the
+   * last field.
+   * See bug 1113134 for related explanation.
+   */
+  function getNextTemplateId(type, container) {
```
Comment (ferjm) [inline apps/communications/contacts/js/views/form.js]: `type` is unused ;)

Comment (arcturus) [inline apps/communications/contacts/js/views/form.js]: Right!

**Comment by try-server-hook** [comment]
[arcturus Francisco Jordano (arcturus)](https://github.com/arcturus) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=93961977355f)

**Comment by try-server-hook** [comment]
[arcturus Francisco Jordano (arcturus)](https://github.com/arcturus) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=666bb8502c36)

**Pull request merged**